src/main.cpp

```diff
@@ -498,23 +509,33 @@ int main ( int argc, char** argv )
 
     if ( !strServerPublicIP.isEmpty() )
     {
-        QHostAddress InetAddr;
-        if ( !InetAddr.setAddress ( strServerPublicIP ) )
+        if ( bIsClient || strCentralServer.isEmpty() )
         {
-            qWarning() << "Server Public IP is invalid. Only plain IP addresses are supported.";
+            qWarning() << "Server Public IP will only take effect when registering a server with a directory server.";
         }
-        if ( strCentralServer.isEmpty() || bIsClient )
+        else
         {
-            qWarning() << "Server Public IP will only take effect when registering a server with a directory server.";
+            QHostAddress InetAddr;
+            if ( !InetAddr.setAddress ( strServerPublicIP ) )
+            {
+                qWarning() << "Server Public IP is invalid. Only plain IP addresses are supported.";
+            }
         }
     }
 
     if ( !strServerBindIP.isEmpty() )
     {
-        QHostAddress InetAddr;
-        if ( !InetAddr.setAddress ( strServerBindIP ) )
+        if ( bIsClient )
+        {
+            qWarning() << "Server Bind IP will only take effect when running a server.";
```

Comment:
... running a server. You're running a client.

Reply:
?

Reply:
We should give the user more information and not let him think too much. Therefore, we should state that this option is not supported since he‘s running a client. Maybe he forgot adding `-s` or something else. See https://github.com/jamulussoftware/jamulus/discussions/1568 and https://uxplanet.org/how-to-write-good-error-messages-858e4551cd4

Reply:
It's not part of this change, though.  I'm just moving stuff around.

Reply:
The message tells the user what the problem is.  I refuse to write messages that take a rough guess as to why the user has caused themselves the problem if there's any chance it could be wrong.  That the program isn't running as a server is true.

They might have remove processing from the code to handle "-s" for all I know and still passed "-s".  No point me trying to guess, is there?